include/boost/geometry/io/wkt/read.hpp

```diff
@@ -82,6 +82,12 @@ struct read_wkt_exception : public geometry::exception
             source += "'";
         }
         complete = message + source + " in '" + wkt.substr(0, 100) + "'";
+        // add extra diagnostic message if the problematic token included
+        // common invalid whitespace characters to help users
+        if (it->find_first_of("\t\r\n") != std::string::npos)
```

Comment:
I'm ok with this approach but I'd like to present different ones and get your feedback. What do you think about the following alternatives:

1. Check wider range of invalid characters because this might potentially help other users, not only the ones dealing with whitespaces and add this specific character to the additional info note below.

2. Escape special characters in `wkt.substr(0, 100)` in the message instead of adding the note, e.g. put tab as `\t`.